specs.json

```diff
@@ -417,6 +420,13 @@
     ]
   },
   "https://tc39.es/proposal-temporal/",
+  {
+    "url": "https://tc39.es/source-map-spec/",
```

Comment:
This should replace the existing sourcemaps spec entry.

Reply:
Ah right, I had forgotten about this one...

Reply:
I propose to reuse the older entry, updating just the URL. The shortname will likely change in the future (when TC39 gives the spec an `ecma-xxx` name) in any case, and this avoids having to track the former shortname for the benefits of Specref.